src/auth/bearer/fake/fake.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2014, Treehouse Networks Ltd. New Zealand
```

Comment:
Sounds good, but you forgot to remove the copyright statement from this file.

I continue to object to this statement because I do not want to be left with a sad choice of either adding "my" statements to every file I touch or being treated as a second class citizen while adding code to files that have statements from "others". I thought we have already discussed all this and decided that there will be no custom copyright statements in Squid files by default. I do not understand why you insist on adding such a custom copyright statement here. AFAICT, it is not needed and it violates the agreed upon policy. Please remove it.